model/config.go

```diff
@@ -481,6 +507,71 @@ func (store *istioConfigStore) Policy(instances []*ServiceInstance, destination
 	return out
 }
 
+// DefaultProxyConfig for individual proxies
+func DefaultProxyConfig() proxyconfig.ProxyConfig {
+	return proxyconfig.ProxyConfig{
+		ConfigPath:             "/etc/istio/proxy",
+		BinaryPath:             "/usr/local/bin/envoy",
+		ServiceCluster:         "istio-proxy",
+		DrainDuration:          ptypes.DurationProto(2 * time.Second),
+		ParentShutdownDuration: ptypes.DurationProto(3 * time.Second),
```

Comment:
Since we no longer need such fast draining for tcp proxying, lets bump this to sane values of 30, 45. The only thing changing is ingress certs and mTLS certs. Shorter drain durations lead to more segfaults due to races. We should probably also restrict the agent to do just one reload at a time (i believe we do that already)

Reply:
Ok